tests/Tests/Search/SearchTemplate/SearchTemplateApiTests.cs

```diff
@@ -101,13 +101,6 @@ public class SearchTemplateInvalidApiTests
 			(c, r) => c.SearchTemplateAsync<Project>(r)
 		);
 
-		protected override void ExpectResponse(ISearchResponse<Project> response)
-		{
-			response.ServerError.Should().NotBeNull();
-			if (TestConfiguration.Instance.ElasticsearchVersion < "7.7.0")
-				response.ServerError.Error.Reason.Should().Contain("no [query]");
-			else
-				response.ServerError.Error.Reason.Should().Contain("unknown query [atch]");
-		}
+		protected override void ExpectResponse(ISearchResponse<Project> response) => response.ServerError.Should().NotBeNull();
```

Comment:
maybe

```c#
response.ServerError.Error.Reason.Should().Contain("query");
```

for both? It would provide a slightly stronger assertion for why there is an error

Reply:
I was just conscious that binding the test to the error message probably isn't a good idea as the error message can change.

Reply:
it's a fair point, historically verifiable! Is there some other way we can assert that the error is related to what we expect it to be?